content/apps/oauth2-scopes.md

```diff
@@ -0,0 +1,52 @@
+# OAuth scopes
+
+During App authorization, App must specify the minimal set of scopes that is required for the App to function.
```

Comment:
```suggestion
Part of the app authorization process requires specifying which parts of a PIM's data the app needs access to.
```

Reply:
needs != requires

In the OAuth spec, you request scopes you need and the Authorization Server decides which ones will be given to you.
You can totally imagine that each requested scope can be checked/unchecked by the user granting you access.
That's why you have to check, in the Access Token Response which scopes were granted to your App.

An App cannot *requires* scopes to work.

Reply:
@willy-ahva https://shopify.dev/api/usage/access-scopes